backend/src/main/java/wooteco/retrospective/infrastructure/auth/JwtTokenProvider.java

```diff
@@ -0,0 +1,44 @@
+package wooteco.retrospective.infrastructure.auth;
+
+import io.jsonwebtoken.*;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.Date;
+
+@Component
+public class JwtTokenProvider {
+
+    @Value("${security.jwt.token.secret-key}")
+    private String secretKey;
+    @Value("${security.jwt.token.expire-length}")
+    private long validityInMilliseconds;
+
+    public String createToken(String payload) {
+        Claims claims = Jwts.claims().setSubject(payload);
+        Date now = new Date();
+        Date validity = new Date(now.getTime() + validityInMilliseconds);
+
+        return Jwts.builder()
+                .setClaims(claims)
+                .setIssuedAt(now)
+                .setExpiration(validity)
+                .signWith(SignatureAlgorithm.HS256, secretKey)
+                .compact();
+    }
+
+    public String getPayload(String token) {
+        return Jwts.parser().setSigningKey(secretKey).parseClaimsJws(token).getBody().getSubject();
+    }
```

Comment:
```suggestion
    public String getPayload(String token) {
        validateToken(token);
        return Jwts.parser().setSigningKey(secretKey).parseClaimsJws(token).getBody().getSubject();
    }
```

여기 메소드에도 `validateToken()`이 필요하다고 생각해요 !

Reply:
@da-nyee 책임이 다르다 생각하는데 어떻게 생각하시나요?? 정책에 따라 다르겠지만 token이 유효하지 않더라도 payload는 봐야 할 경우가 있다고 생각합니다!

Reply:
애초에 토큰이 유효하지 않으면 payload를 볼 수 없지 않나요?